templates/csv.yaml

```
@@ -144,7 +144,7 @@ spec:
     Knative Functions bring greater efficiency, more scalability and faster development to facilitate rapid go-to-market.
 
     Other features include:
-    - Build strategies including Source-to-Image (S2I) and Buildpacks
+    - Buildpacks
```

just a nit -  this couldve been
```suggestion
    - Build strategy Buildpacks
```
but dont mind either way
/lgtm